nunaliit2-couch-command/src/main/java/ca/carleton/gcrc/couch/command/AtlasProperties.java

```diff
@@ -102,52 +102,25 @@ static public AtlasProperties fromProperties(Properties props) throws Exception
 
 	static public void readProperties(File atlasDir, Properties props) throws Exception {
 		// install.properties
-		{
-			File installPropFile = new File(atlasDir,"config/install.properties");
-			if( installPropFile.exists() && installPropFile.isFile() ){
-				FileInputStream fis = null;
-				try {
-					fis = new FileInputStream(installPropFile);
-					InputStreamReader reader = new InputStreamReader(fis,"UTF-8");
-					props.load(reader);
-
-				} catch(Exception e) {
-					throw new Exception("Unable to read config properties from: "+installPropFile.getAbsolutePath(), e);
-
-				} finally {
-					if( null != fis ){
-						try{
-							fis.close();
-						} catch(Exception e) {
-							// Ignore
-						}
-					}
-				}
-			}
-		}
+		File installPropFile = new File(atlasDir,"config/install.properties");
+		readConfigFile(installPropFile, props);
 
 		// sensitive.properties
-		{
-			File sensitivePropFile = new File(atlasDir,"config/sensitive.properties");
-			if( sensitivePropFile.exists() && sensitivePropFile.isFile() ){
-				FileInputStream fis = null;
-				try {
-					fis = new FileInputStream(sensitivePropFile);
-					InputStreamReader reader = new InputStreamReader(fis,"UTF-8");
-					props.load(reader);
-
-				} catch(Exception e) {
-					throw new Exception("Unable to read config properties from: "+sensitivePropFile.getAbsolutePath(), e);
+		File sensitivePropFile = new File(atlasDir,"config/sensitive.properties");
+		readConfigFile(sensitivePropFile, props);
+	}
 
-				} finally {
-					if( null != fis ){
-						try{
-							fis.close();
-						} catch(Exception e) {
-							// Ignore
-						}
-					}
-				}
+	static public void readConfigFile(File configFile, Properties props) throws Exception {
+		FileInputStream fis = null;
+		try {
+			fis = new FileInputStream(configFile);
+			InputStreamReader reader = new InputStreamReader(fis,"UTF-8");
+			props.load(reader);
+		} catch(Exception e) {
+			throw new Exception("Unable to read config properties from: " + configFile.getAbsolutePath(), e);
+		} finally {
+			if( null != fis ){
+				fis.close();
```

Comment:
The old code did try catch when trying to close the file input stream, probably want to keep that here as well.